wagtail/blocks/list_block.py

```diff
@@ -224,6 +225,32 @@ def clean(self, value):
 
         return ListValue(self, bound_blocks=result)
 
+    def normalize(self, value):
+        if isinstance(value, ListValue):
+            return value
+
+        # Squint a little and see if it looks like a list
```

Comment:
It might be nice to make this (and `StreamBlock.normalize`) generic over iterable types, rather than requiring a `list`?